.rhcicd/pr_check.sh

```diff
@@ -36,3 +36,13 @@ cat << EOF > ${WORKSPACE}/artifacts/junit-dummy.xml
     <testcase classname="dummy" name="dummytest"/>
 </testsuite>
 EOF
+
+# adding floorist tests.
+export COMPONENT_NAME="policies-engine"
```

Comment:
Lets not override component name. Is this to hack this around to have a second CJI run?

Reply:
This is how I was able to get the pod to deploy. This is what I had to use to get the pod to deploy when I was running  deploy-iqe-cji manually. I do not believe it will deploy otherwise.

Reply:
This has to be set to the component that IQE will use in the CJI to lookup the clowdenv configmap

Reply:
You will also notice that the existing line 26 has this: export COMPONENTS_W_RESOURCES="policies-engine".

Reply:
I am testing removing line 41 in a other PR. I'll make the change here if it passes.

Reply:
I tried removing line 41 but the tests time out attempting to deploy via cji. I ran into this when I was deploying and testing in ephemeral manually. You will notice that in a successful run here we use "bonfire deploy-iqe-cji policies-engine" to deploy resources to run the floorist tests. When I remove line 41 (export COMPONENT_NAME="policies-engine")    "bonfire deploy-iqe-cji policies-ui-backend" is used and the deployment times out. You can see this here:  https://ci.ext.devshift.net/job/RedHatInsights-policies-ui-backend-pr-check/705/console

Reply:
This needs to be set to the same component name, otherwise it won't work as it wouldn't get the right credentials to the database. If we're going to have this running in other projects, not all have two (or more) components. So we don't have that luxury to hack it around with using a different component.

Can you please check what is causing the timeout? I see that it does want to reuse the same test pod. Is there a way to force it to use a new one with the new name?

Reply:
@prichard77 try to utilize option `-c` (`--cji-name`) for the CJI deploy command. Example: `--cji-name floorist-iqe`

Reply:
I added support to pass in cji name and also I added support for multiple floorplans.

Reply:
The `COMPONENT_NAME` is still wrong. Please remove this line.

Reply:
```
(.iqe-env) ➜  iqe bonfire deploy-iqe-cji policies-ui-backend --cji-name iqe-floorist-tests --marker floorist_smoke --plugins floorist --env-var "FLOORPLANS_TO_TEST=policies-backend-hms" --image-tag floorist --env clowder_smoke --namespace $NAMESPACE

================= 1 passed, 1 deselected, 4 warnings in 0.63s ==================

(.iqe-env) ➜  iqe bonfire deploy-iqe-cji policies-engine --cji-name iqe-floorist-tests-engine --marker floorist_smoke --plugins floorist --env-var "FLOORPLANS_TO_TEST=policies-backend-hms" --image-tag floorist --env clowder_smoke --namespace $NAMESPACE

================= 1 passed, 1 deselected, 4 warnings in 0.66s ==================
```

Neither is wrong. This controls the clowdapp that owns the CJI, not a specific component cdappconfig that IQE will have access to, or what the IQE pod is testing against.